docs: Troubleshoot incompatible docling-serve version (#1511)
langflow-ai/openrag: docs/docs/support/troubleshoot.mdx

* Make sure you have enough free space (more than 50 GB) available for the temporary storage required for document ingestion.
* Clear your browser's cache.

## OpenRAG reports Docling Serve errors but docling-serve is running

Make sure your `docling-serve` version is compatible with your OpenRAG version.
OpenRAG version 0.4.1 requires `docling-serve` version 1.5.0.

> If you installed a different version of `docling-serve` before starting or installing OpenRAG, you can do any of the following:

Author: ```suggestion
If you started a `docling-serve` container before starting or installing OpenRAG, and the `docling-serve` version is incompatible with your OpenRAG version, you need to remove or restart the incompatible container.
There are several ways to do this:
```

> * Remove the incompatible version of `docling-serve`, and then [reinstall OpenRAG](/reinstall) with `uv` or `uvx` to automatically install the compatible version.

Reviewer: Remove the incompatible version
Might clarify a little here. I'm removing the docling-serve container, reinstalling openrag, then starting a new docling-serve container?

Author: yes. I'll clarify that.

Author: uv/uvx will auto-start the container. The problem is if you already have one running, openrag can't auto-start with the expected version.

> * Remove the incompatible version of `docling-serve`, and then [reinstall OpenRAG](/reinstall) with `uv` or `uvx` to automatically install the compatible version.

Author: ```suggestion
* Remove the `docling-serve` container that is running the incompatible version, and then [reinstall OpenRAG](/reinstall) with `uv` or `uvx` to automatically start a compatible `docling-serve` container.
```
